src/index.js

```diff
@@ -6,11 +6,17 @@ import template from 'babel-template';
  */
 const buildHelper = template(`
     function HELPER(cls){
+        var setPrototypeOf = Object.setPrototypeOf || (function setPrototypeOf(a, b) {
+            a.__proto__ = b;
+        });
+        var getPrototypeOf = Object.getPrototypeOf || (function getPrototypeOf(a)) {
+            return a.__proto__;
```

Comment:
I can't imagine a case where `getPrototypeOf` doesn't exist but `__proto__` does. `getPrototypeOf` is from ES5, so it either exists, or there's no way to polyfill it AFAIK.